javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -839,6 +839,59 @@ public NoSuchElementException(String message) {
         return result;
     }
 
+    /**
+     * Get the element {@code executableElement} of the annotation {@code anno}, where the element
```

Comment:
There is too much duplication between this method and the one above.  Please abstract the common code into a helper method that both can call.